src/app.ts

```diff
@@ -191,7 +191,8 @@ const MESSAGE_MAP: { [key in ResultStatus]: string[] } = {
     'Everything is ship-shape!',
     'Ex-clam-ation point!',
     'Ac-clam-ations!',
-    'Fin-tastic job!'
+    'Fin-tastic job!',
+    'You dolphinitely have the hang of this!'
```

Comment:
Unfor-tuna-tely, I think this line is slightly too long, so it ends up looking like this:
![image](https://user-images.githubusercontent.com/52747882/91559779-b0974280-e906-11ea-9923-23b9efd917ea.png)

It looks like it is 3 characters too long.

Reply:
We could do "You're dolphinitely doing great", or "You dolphinitely got it!", or something like that. Or "You dolphinitely got the hang of it!" to stay close to the original pun.

Reply:
Oh good call. Somehow I like "You're dolphinitely doing great!" so I think I'll go with that!